migrations/Version202211031203542260_taoQtiTest.php

```diff
@@ -20,9 +20,17 @@ public function getDescription(): string
 
     public function up(Schema $schema): void
     {
-        class_alias(TestPackageMetadataExport::class, 'oat\\taoQtiTest\\models\\export\\metadata\\TestMetadataByClassExportHandler');
-        class_alias(TestPackage2p1Export::class, 'taoQtiTest_models_classes_export_TestExport');
-        class_alias(TestPackage2p2Export::class, 'taoQtiTest_models_classes_export_TestExport22');
+        if(!class_exists('oat\\taoQtiTest\\models\\export\\metadata\\TestMetadataByClassExportHandler')) {
```

Comment:
Probably because this is also added there https://github.com/oat-sa/extension-tao-testqti/blob/develop/models/classes/export/Formats/Metadata/TestPackageExport.php#L62 🤦 

To be honest, I do not know why we decided to use aliases instead of keep the other classes extending the new ones and mark them as deprecated or so...

Thanks @hectoras